### DISCUSSION TITLE
Are Flatlist optimization props/strategies still relevant when using Flashlist?

### DISCUSSION THREAD

**Discussion by TowhidKashem** [Shopify/flash-list#552] [General]
Hi,

I converted all my app's Flatlists -> Flashlist and the performance has dramatically improved, thanks! Now I'm just wondering if I need to keep the old performance improvement props I had on Flatlist in Flashlist as well?

```
    maxToRenderPerBatch: 5, // 10 (smaller the better, more means JS thread is busier and causes response lag, too small means blank space)
    updateCellsBatchingPeriod: 25, // 50 (smaller the better, too much means responsiveness lag, too small means blank space)
    initialNumToRender: 5, // 10 (smaller better, too small increased chance of blank space)
    windowSize: 11, // 21 (smaller the number more memory saved, bigger the number less chances of seeing blank space)
```

Are these still relevant? Does Flashlist still make use of these or can I just ditch em?

**Comment by naqvitalha** [comment]
You can ditch them. They're not used. 
Also, I'm glad that FlashList helped :)

**Reply by fortmarek** [reply]
You can read more about how to optimize components for `FlashList` [here](https://shopify.github.io/flash-list/docs/fundamentals/performant-components) – but as Talha mentioned, those props you mention are no longer needed.
